capstone_project/src/main/webapp/styles/posts.css

```diff
@@ -21,36 +106,22 @@
   background-color: rgba(0, 0, 0, 0);
 }
 
-.badge {
-  background-color: red;
+.reaction-counter {
+  position: relative;
+  left: 20px;
+  bottom: 50px;
+  color: #fff;
+  background-color: #49c4d4;
+  margin: 0 -0.8em;
```

Comment:
Are these attributes usually in alphabetical order? Just curious about convention!

Reply:
Yes, I must have missed these thanks